src/lib/Scenes/Collection/Components/__tests__/FeaturedArtists-tests.tsx

```diff
@@ -174,6 +177,18 @@ describe("FeaturedArtists", () => {
       expect(output).toContain("Kenny Scharf")
     })
 
-    // it("tracks an event when 'View more' is tapped", () => {})
+    it("tracks an event when 'View more' is tapped", async () => {
+      const tree = await render(CollectionFixture)
+      const viewMore = tree.find("EntityHeader").filterWhere(x => x.text().includes("View more"))
+
+      viewMore.simulate("click")
+
+      expect(Events.postEvent).toHaveBeenCalledWith({
+        action_name: "viewMore",
+        context_module: "FeaturedArtists",
+        context_screen: "Collection",
+        flow: "FeaturedArtists",
```

Comment:
This adds a test for the one brand new tracking event introduced in this PR.

I fought with trying to track the Collection `screenview` event for a bit, but gave up 😭